dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ORCIDHelper.java

```diff
@@ -85,6 +88,8 @@ public static void init(DockstoreWebserviceConfiguration configuration) {
             baseUrl = orcidAuthUrl.getProtocol() + "://" + orcidAuthUrl.getHost() + "/";
             // baseApiUrl should result in something like "https://api.sandbox.orcid.org/v3.0/" or "https://api.orcid.org/v3.0/"
             baseApiUrl = orcidAuthUrl.getProtocol() + "://api." + orcidAuthUrl.getHost() + "/v3.0/";
+            // if hitting the public API, no point in using the sandbox public API
```

Comment:
It seems like the mixture of environments could potentially cause some confusion, at least theoretically. But the confusion would only be in staging/qa, and would probably be weird edge cases.